docs/docs/build/packages.mdx

```diff
@@ -24,8 +24,8 @@ contained within itself, as well as to inspect the state of the collaboration se
 You'll use one or more shared objects in your container to model your collaborative data. The `fluid-framework` package includes
 three data structures that cover a broad range of scenarios:
 
-1. [SharedMap](../data-structures/map), a map-like data structure for storing key/value pair data.
-2. [SharedString](../data-structures/string), a data structure for string data.
+1. [SharedMap](../data-structures/map.mdx), a map-like data structure for storing key/value pair data.
+2. [SharedString](../data-structures/string.md), a data structure for string data.
```

Comment:
should this me .mdx?

Reply:
Nvm, i see that the string file is a .md. Wondering why its not mdx like the right?

Reply:
We have a few documents that are still `.md` because they use our `markdown-magic` tooling to embed contents from other Markdown docs elsewhere in the repo. That tool, unfortunately, doesn't work in `.mdx`.

We have an item in the backlog to explore alternatives to `markdown-magic` that would suit our needs here, but until then such documents have to remain `.md`.